templates/practice/documentation.html

```diff
@@ -49,16 +49,13 @@ <h1>Documentation at Canonical</h1>
           </div>
           <ul>
             <li class="p-side-navigation__item">
-              <a class="highlight-link is-active" href="#introduction">Introduction</a>
+              <a class="highlight-link" href="#documentation-as-practice">Documentation as practice</a>
```

Comment:
This navigation button doesn't do anything

Reply:
It has to be there for it to mark it as we scroll